packages/block-library/src/buttons/edit.js

```diff
@@ -15,6 +15,17 @@ import { name as buttonBlockName } from '../button';
 
 const ALLOWED_BLOCKS = [ buttonBlockName ];
 
+const DEFAULT_BLOCK = {
+	name: buttonBlockName,
+	attributesToCopy: {
+		backgroundColor: true,
+		className: true,
+		gradient: true,
+		textColor: true,
+		width: true,
```

Comment:
Do we also want to copy border and spacing styles?

Reply:
Oh, yes, and font settings, too! I totally forgot about those attributes provided by block supports. I've added a few more here and it seems to be working well:

![Kapture 2022-01-19 at 16 52 11](https://user-images.githubusercontent.com/14988353/150072223-c69e71d6-7634-426e-87f8-64bbc15d2b57.gif)